README.md

```diff
@@ -40,6 +40,9 @@ The meetings are independent of any organization. However, Danny Ryan & Tim Beik
 
 | №   | Date                                 | Agenda                                              | Notes                                                                                                                                                                    | Recording                                                                          |
 | --- | ------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| 192 | July 18, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1098) | [notes](AllCoreDevs-EL-Meetings/Meeting 192.md)\| [Twitter](https://x.com/TimBeiko/status/1813997958939267104)      |[Video](https://youtube.com/live/kL58hvM0E68) |
+| 191 | July 4, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1080) | [notes](AllCoreDevs-EL-Meetings/Meeting 191.md)\| [Twitter] | [Video](https://youtube.com/live/58_bJD_dmm0) |
+| 190 | July 4, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1066) | [notes](AllCoreDevs-EL-Meetings/Meeting 190.md)\| [Twitter] | [Video](https://youtube.com/live/8VGf-EE6zNE) |
```

Comment:
```suggestion
| 190 | June 20, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1066) | [notes](AllCoreDevs-EL-Meetings/Meeting 190.md)\| [Twitter] | [Video](https://youtube.com/live/8VGf-EE6zNE) |
```

Reply:
I missed this earlier, apologies. Please update, thanks!

Reply:
> I missed this earlier, apologies. Please update, thanks!

Done